website/docs/d/scc_posture_collectors.html.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_collectors"
+description: |-
+  Get information about list_collectors
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_collectors
+
+Provides a read-only data source for list_collectors. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_scc_posture_collectors" "list_collectors" {
+}
+```
+
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the list_collectors.
+* `collectors` - (Required, List) The array of items returned.
+Nested scheme for **collectors**:
+	* `approved_internet_gateway_ip` - (Optional, String) The approved internet gateway ip of the collector. This field will be populated only when collector is installed.
```

Comment:
In computed attributes we can have required/optional.. Remove from all 
`approved_internet_gateway_ip` - (String)

Reply:
Code modified as per review comments